qutebrowser/browser/commands.py

```diff
@@ -514,14 +514,47 @@ def tab_clone(self, bg=False, window=False):
         return newtab
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def tab_detach(self):
```

Comment:
I think it's fine to remove `:tab-detach` entirely without having an alias for it FWIW (but I'd like to hear other's opinions).

While `:tab-give` is slightly less intuitive than `:tab-detach`, I don't think we need two commands which essentially do the same thing, and with v1.0.0 we can do breaking changes like this one without having to worry about backwards compatibility.

Reply:
I re-added it as deprecated command in f1ddf58260936ecdcb2932997d5ac8955ebc6ee8 as it was straightforward to do so and will probably save me some "my keybindings don't work anymore!" questions.